### PR TITLE
Add information about simplecov and json lock to readme

### PR DESCRIPTION
**Pull request by ABaldwinHunter**
@codeclimate/review 

Alternative to https://github.com/codeclimate/ruby-test-reporter/pull/119

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,24 @@ Add the following to your spec or test helper:
...
+### Gem::InstallError: json requires Ruby version ~> 2.0
+
+Some versions of simplecov after 0.11.2 effectively don't support ruby 1.9.3
+due to a loose json dependency that picks the latest version of `json`.
+
+See full explanation of issue: [colszowka/simplecov#511](https://github.com/colszowka/simplecov/issues/511)
+
+The Code Climate Ruby test reporter supports ruby > 1.9. To run with ruby less
+than 2.0, you may need to specify a locked dependency to simplecov 0.11.2 or
+json < 2.0 in your project's gem file.
```
Comment (GordonDiggs) [inline README.md]: 💄 typo: `gem file` (should not have a space).

Additionally, the example below is a gemspec file - I think a Gemfile example would be better

File: README.md [hunk 2]
```diff
@@ -135,6 +135,20 @@ Add the following to your spec or test helper:
...
+due to a loose json dependency that picks the latest version of `json`.
+
+See full explanation of issue: [colszowka/simplecov#511](https://github.com/colszowka/simplecov/issues/511)
+
+The Code Climate Ruby test reporter supports ruby > 1.9. To run with ruby less
+than 2.0, you may need to specify a locked dependency to simplecov 0.11.2 or
+json < 2.0 in your project's gemfile:
+
+          gem "codeclimate-test-reporter"
+          gem "simplecov", "~> 0.11.2" #<--only here to lock
```
Comment (wfleming) [inline README.md]: Personally I find this comment more confusing than enlightening.

**Comment by wfleming** [comment]
Small note, take it or leave it. LGTM.

File: README.md (hunk 2 above)
Comment (ABaldwinHunter) [inline README.md]: Thanks, updated!

**Comment by ABaldwinHunter** [comment]
closes https://github.com/codeclimate/ruby-test-reporter/pull/119

**Pull request merged**